runtime/v2/builder.go

```diff
@@ -70,7 +74,7 @@ func (a *AppBuilder[T]) RegisterStores(keys ...string) {
 }
 
 // Build builds an *App instance.
-func (a *AppBuilder[T]) Build(opts ...AppBuilderOption[T]) (*App[T], error) {
+func (a *AppBuilder[T]) Build(viper *viper.Viper, opts ...AppBuilderOption[T]) (*App[T], error) {
```

Comment:
It is already supplied in depinject, we should just store that on the app struct directly and get it here.